/**:
  ros__parameters:
    lookahead_gain: 1.0
    lookahead_min_distance: 1.0
    speed_proportional_gain: 1.0
    use_external_target_vel: false
    external_target_vel: 1.0
